src/theme/index.ts

```diff
@@ -78,7 +82,7 @@ export const useThemeable = <
             // @ts-ignore
             if (value && t.overrides[index]) {
               // @ts-ignore
```

Comment:
ignore still needed?

Reply:
Yes, this is the mess of a method I told andres to have fun refactoring... I'll probably get around to it eventually